SPECS/unzip/CVE-2021-4217.patch

```diff
@@ -0,0 +1,51 @@
+From 24bfe051b63f7347d06d852a277ceb657be5d1d4 Mon Sep 17 00:00:00 2001
+From: archana25-ms <v-shettigara@microsoft.com>
+Date: Tue, 25 Mar 2025 18:05:10 +0000
+Subject: [PATCH] Address CVE-2021-4217
+Upstream Patch Reference: https://bugs.launchpad.net/ubuntu/+source/unzip/+bug/1957077
```

Comment:
Patch Link: https://[launchpadlibrarian.net/580782282/0001-Fix-null-pointer-dereference-and-use-of-uninitialized-data.patch](https://launchpadlibrarian.net/580782282/0001-Fix-null-pointer-dereference-and-use-of-uninitialized-data.patch)